README.md

```diff
@@ -37,9 +37,12 @@ spec:
         http:
           paths:
           - backend:
```

Comment:
does `- backend:` suppose to be empty ? 
I wouldn't care but in README.md,  for easier orientation we can leave `path` as the first item https://github.com/k8gb-io/k8gb/pull/854/files#diff-524f9815d92e58f96cf2ca5d5cb3b2457e7dbf58b7b16d05b10bb2551eb1d041R30-R35

Reply:
good catch, should be fixed now